lib/provider/camunda/parts/implementation/ErrorEventDefinition.js

```diff
@@ -36,29 +35,52 @@ module.exports = function(
       id: 'error-element-message',
       label: translate('Error Message'),
       referenceProperty: 'errorRef',
-      modelProperty: 'errorMessage'
+      modelProperty: 'errorMessage',
+      description: translate('of the global Error')
     })
   );
 
   if (showErrorCodeVariable) {
```

Comment:
I believe test cases (should show / not show / set / get / validate) for errorCodeVariable and errorMessageVariable are missing